src/main/java/org/webonise/polygon/Application.java

```diff
@@ -0,0 +1,118 @@
+package org.webonise.polygon;
+
+import java.util.InputMismatchException;
+import java.util.Scanner;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class Application {
+    private final PolygonCalculator polygonCalculator;
+    private final Scanner input;
+    private static final int MIN_SIDES = 3;
+    private static final int MAX_SIDES = 10;
+    private final Logger logger;
+
+    public Application() {
+        logger = Logger.getLogger(Application.class.getName());
+        polygonCalculator = new PolygonCalculator();
+        input = new java.util.Scanner(System.in);
+    }
+
+    public void start() {
+
+        try {
+            startApplication();
+        } catch (IllegalArgumentException illegalArgumentException) {
+            logger.log(Level.INFO, illegalArgumentException.getMessage());
+        } catch (InputMismatchException inputMismatchException) {
+            logger.log(Level.INFO, inputMismatchException.getMessage());
+        }
+    }
+
+    private void startApplication() throws IllegalArgumentException, InputMismatchException {
+
+        logger.log(Level.INFO, "========Polygon Calculator========");
+
+        logger.log(Level.INFO, "enter no of sides(int) of Polygon: ");
+        int noOfSides = input.nextInt();
+
+        checkNoOfSideLimit(noOfSides);
+
+        logger.log(Level.INFO, "enter length of side(double) of Polygon: ");
+        double sideLength = input.nextDouble();
+
+        logger.log(Level.INFO, "Menu \n\t1) Perimeter \n\t2) Sum of Interior Angles");
+        logger.log(Level.INFO, "Enter option(1 or 2): ");
+        int option = input.nextInt();
+
+        switch (option) {
+            case 1:
+                perimeterMenu(noOfSides, sideLength);
+                break;
+
+            case 2:
+                sumOfInteriorAnglesMenu(noOfSides, sideLength);
+                break;
+
+            default:
+                throw new IllegalArgumentException("Entered Invalid Option");
+        }
+    }
+
+    private void checkNoOfSideLimit(int noOfSides) throws IllegalArgumentException {
+
+        if (noOfSides < MIN_SIDES || noOfSides > MAX_SIDES) {
+            throw new IllegalArgumentException("Error: No of Sides should be >= " + MIN_SIDES + " and " + "<= " + MAX_SIDES);
+        }
+    }
+
+    private void perimeterMenu(int noOfSides, double sideLength) throws IllegalArgumentException, InputMismatchException {
+
+        logger.log(Level.INFO, "==========Perimeter==========");
+        logger.log(Level.INFO, "Calculating Permeter by \n\t1)passing value  2) by polygon object");
```

Comment:
No need to give a choice here. Whatever you need to calculate perimeter you already have it.